### PR TITLE
Enable config reloading on SIGHUP

### PR DESCRIPTION
**Pull request by fmoor**
Fixes https://github.com/edgedb/edgedb/issues/4278

### CODE REVIEW

File: edb/server/main.py [hunk 1]
```diff
@@ -246,6 +246,15 @@ async def _run_server(
...
         )
 
+        def load_configuration(signum, frame):
+            logger.info("reloading configuration")
+            ss.get_loop().call_soon_threadsafe(
+                ss.load_tls, args.tls_cert_file, args.tls_key_file)
+            ss.get_loop().call_soon_threadsafe(
+                ss.load_jwcrypto, args.jws_key_file, args.jwe_key_file)
+
+        signal.signal(signal.SIGHUP, load_configuration)
```
Comment (fmoor) [inline edb/server/main.py:256]: I think this overrides https://github.com/edgedb/edgedb/blob/master/edb/server/daemon/daemon.py#L208. :thinking:

Comment (1st1) [reply edb/server/main.py:256]: Yeah, we should call the original handler too

Comment (1st1) [reply edb/server/main.py:256]: (to enable log rotation)

Comment (fmoor) [reply edb/server/main.py:260]: changed

**Review by elprans — APPROVED** [review]

**Pull request merged**